### PR TITLE
Spelling checks

### PR DESCRIPTION
**Pull request by MvdO79**
https://github.com/cfwheels/cfwheels/issues/1159

### CODE REVIEW

File: wheels/tests/view/text/excerpt.cfc [hunk 1]
```diff
@@ -26,7 +26,7 @@ component extends="wheels.tests.Test" {
 		args.phrase = "excerpt view helper";
 		args.radius = "10";
 		e = _controller.excerpt(argumentcollection = args);
-		r = "[more]sting the excerpt view helper to see if[more]";
+		r = "[more]testing the excerpt view helper to see if[more]";
```
Comment (bpamiri) [inline wheels/tests/view/text/excerpt.cfc:29]: @MvdO79 can you revert this one. This is causing this test to fail which is failing the PR.

**Review by bpamiri — CHANGES_REQUESTED** [review]

**Comment by MvdO79** [comment]
All tests are passed now!

**Review by bpamiri — APPROVED** [review]

**Comment by bpamiri** [comment]
Awesome work @MvdO79. This was a huge bunch of changes.

**Comment by MvdO79** [comment]
Glad to help!

**Pull request merged**